qdrant-landing/content/documentation/embeddings/premai.md

```diff
@@ -0,0 +1,477 @@
+---
+title: Prem AI
+weight: 1000
+aliases: [ ../integrations/premai/ ]
+---
+
+# Prem AI
+
+[PremAI](https://premai.io/) is an all-in-one platform that simplifies the creation of robust, production-ready applications powered by Generative AI. By streamlining the development process, PremAI allows you to concentrate on enhancing user experience and driving overall growth for your application. You can quickly start using our platform [here](https://docs.premai.io/quick-start).
+
+Qdrant is compatible with Prem AI SDK (both Python and Javascript interfaces). We start by installing the SDKs. Here is how you install premai Python SDK. 
+
+```bash
+pip install premai
+```
+
+To install the npm package, type this command:
+
+```bash
+npm install @premai/prem-sdk 
+```
+
+### Import all required packages
+
+We start by importing all our required packages. Here is how you do that for python
+
+```python
+import os 
+import getpass
+from premai import Prem
+
+from qdrant_client import QdrantClient
+from qdrant_client.models import Distance, VectorParams
+
+if os.environ.get("PREMAI_API_KEY") is None:
+    os.environ["PREMAI_API_KEY"] = getpass.getpass("PremAI API Key:")
+```
+
+And here is how you do for javascript
+
+```jsx
```

Comment:
Please have the snippets together with `python` and `typescript` labels similar to the other doc pages.
We can then have the language switch buttons.

Reply:
Done